### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem
- *New version upstream*

## Solution
- *Upgrade to v 2.0.10*
- *Remove rm -rf #13*
- *Set new permissions*

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Package_check results
---
*If you have access to [App Continuous Integration for packagers](https://yunohost.org/#/packaging_apps_ci) you can provide a link to the package_check results like below, replacing '-NUM-' in this link by the PR number and USERNAME by your username on the ci-apps-dev. Or you provide a screenshot or a pastebin of the results*

[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/webtrees_ynh%20PR14%20(ericgaspar)/)

### CODE REVIEW

File: scripts/upgrade [hunk 1]
```diff
@@ -79,7 +79,7 @@ ynh_setup_source "$final_path"
 cp -a ${final_path}.old/data ${final_path}
 
 # delete temp directory
-rm -Rf ${final_path}.old
+ynh_secure_remove ${final_path}.old
```
Comment (yalh76) [inline scripts/upgrade:82]:
```suggestion
ynh_secure_remove --file=${final_path}.old
```

**Review by yalh76 — CHANGES_REQUESTED** [review]

**Review by yalh76 — APPROVED** [review]

File: scripts/remove [hunk 2]
```diff
@@ -76,4 +76,4 @@ ynh_system_user_delete --username=$app
 # END OF SCRIPT
 #=================================================
 
-ynh_script_progression --message="Removal of $app completed"
+ynh_script_progression --message="Removal of Webtrees completed"
```
Comment (anmol26s) [inline scripts/remove:79]: Will this not provide the app instance name here? What if the user have multiple instances?And he will not be given with correct instance name.

Comment (ericgaspar) [reply scripts/remove]: Point taken -> d428b3a

**Comment by ericgaspar** [comment]
Maybe the registration password process should be changed for this app:
For the moment, the password is generated randomly by the app and sent to the administrator by e-mail.

**Two problems:**
- A user [issued](https://forum.yunohost.org/t/packagers-this-app-is-still-using-the-skipped-protected-unprotected-uris-regex-settings/13252/7) that the e-mail with the password does not reach his mailbox and therefore cannot log in.
What about letting the user choose their password during the installation process?
- Users will also complain that the password is sent in clear by e-mail.
If the password is chosen by the administrator, do we need to send an e-mail?

**Pull request merged**